administrator/components/com_media/resources/scripts/components/browser/items/video.vue

```diff
@@ -5,8 +5,20 @@
     @mouseleave="hideActions()"
   >
     <div class="media-browser-item-preview">
-      <div class="file-background">
-        <div class="file-icon">
+      <div
+        class="file-background"
+        :class="{ 'with-thumbnail': thumbURL }"
+      >
+        <img
+          v-if="thumbURL"
+          class="image-cropped"
+          alt=""
+          :src="thumbURL"
+          :loading="thumbLoading"
+          :width="thumbWidth"
+          :height="thumbHeight"
+        >
+        <div v-if="!thumbURL" class="file-icon">
```

Comment:
```suggestion
        <div v-if="!thumbURL"
          class="file-icon"
        >
```
Fix javascript-cs (5 of 5)